cirq-google/cirq_google/experimental/noise_models/validate_noise_model_from_calibration_test.py

```diff
@@ -0,0 +1,190 @@
+        }}]
+    }}]
+""",
+        v2.metrics_pb2.MetricsSnapshot(),
+    )
+
+    random_seed = 1  # to make sure values match between simulators
+
+    calibration = cirq_google.Calibration(_CALIBRATION_DATA)
+    output_df = validate_noise_model(calibration, seed=random_seed)
+
+    # Construct expected results
+    expected_p00 = np.mean([p00_1, p00_2, p00_3])
+    expected_p11 = np.mean([p11_1, p11_2, p11_3])
+    expected_xeb_fidelity = 1 - np.mean([xeb_1, xeb_2])
+    expected_t1 = np.mean([t1_1, t1_2, t1_3]) * 1000
+
+    num_qubits_xeb = 2
+    N_xeb = 2 ** num_qubits_xeb
+    expected_decay_constant = 1 - (1 - expected_xeb_fidelity) * N_xeb / (N_xeb - 1)
+    N = 2  # single qubit Hilbert space
+    expected_pauli_error = (1 - expected_decay_constant) * (1 - 1 / N ** 2)
+
+    expected_average_error = (1 - expected_decay_constant) * (1 - 1 / N)
+
+    # Create noise model and simulator
+    noise_prop = noise_properties_from_calibration(calibration)
+    noise_model = NoiseModelFromNoiseProperties(noise_prop)
+
+    qubits = [cirq.LineQubit(0), cirq.LineQubit(1)]
+
+    simulator = cirq.sim.DensityMatrixSimulator(noise=noise_model, seed=random_seed)
+
+    # Run experiments
+    estimate_readout = cirq.experiments.estimate_single_qubit_readout_errors(
+        simulator, qubits=[qubits[0]], repetitions=1000
+    )
+
+    xeb_result = cirq.experiments.cross_entropy_benchmarking(
+        simulator, qubits, num_circuits=50, repetitions=1000
+    )
+    measured_xeb = np.mean([datum.xeb_fidelity for datum in xeb_result.data])
+    decay_constant = xeb_result.depolarizing_model().cycle_depolarization
+    t1_results = cirq.experiments.t1_decay(
+        simulator,
+        qubit=qubits[0],
+        num_points=100,
+        repetitions=100,
+        min_delay=cirq.Duration(nanos=10),
+        max_delay=cirq.Duration(micros=1),
+    )
+
+    N = 2  # Dimension of Hilbert Space
+    measured_pauli_error = (1 - decay_constant) * (1 - 1 / N / N)
+    measured_average_error = (1 - decay_constant) * (1 - 1 / N)
+
+    output = []
+
+    output.append(['p00', expected_p00, estimate_readout.zero_state_errors[cirq.LineQubit(0)]])
+    output.append(['p11', expected_p11, estimate_readout.one_state_errors[cirq.LineQubit(0)]])
+    output.append(['XEB Fidelity', expected_xeb_fidelity, measured_xeb])
+    output.append(['T1', expected_t1, t1_results.constant])
+    output.append(['Pauli Error', expected_pauli_error, measured_pauli_error])
+    output.append(['Average Error', expected_average_error, measured_average_error])
+
+    columns = ["Metric", "Initial value", "Measured value"]
+    expected_df = pd.DataFrame(output, columns=columns)
+
+    print(output_df)
+    print(expected_df)
+    print(expected_df['Measured value'], output_df['Measured value'])
+    assert np.allclose(expected_df['Initial value'], output_df['Initial value'], atol=0.5)
+    assert np.allclose(expected_df['Measured value'], output_df['Measured value'], atol=0.5)
```

Comment:
I know I recommended keeping this tolerance broad, but 0.5 may be a bit much given that these are probabilities. Can this be scaled back and still pass reliably?